code/modules/admin/admin_verbs.dm

```diff
@@ -78,6 +78,8 @@ GLOBAL_PROTECT(admin_verbs_admin)
 	/client/proc/fix_say,
 	/client/proc/stabilize_atmos,
 	/client/proc/openTicketManager
+	/client/proc/cmd_admin_rejuvenate,
+	/client/proc/cmd_admin_freeze,
```

Comment:
Can you move it from verbs to player panels? Most of similar verbs (ie ajail) are here.

Reply:
> Can you move it from verbs to player panels? Most of similar verbs (ie ajail) are here.

It's a verb because that's the fastest way to access it, this is intended to be a tool to immediately make the person stop whatever they're doing.

Reply:
Yeah and it's a bad design choice, this option belongs in player panel (which is super easy to access anyway).